recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 88d53bb2eb3ef1d4aae10ee318ddc1e6d9661f5fc80e1c59c6520524c79a16df
 
 build:
-  noarch: python
```

Comment:
Is the removal of noarch really needed?

Reply:
@roederja, unfortunately yes because the `portalocker` dependencies are platform-dependent. As soon as the platform comes into play in the recipe, the recipe can no longer be noarch.